kubernetes/README.md

```diff
@@ -46,16 +46,21 @@ The following steps only apply if you use Google Workspace as your identity prov
 
 1. Follow the steps to [create a Google service account, key, and API client](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client).
 2. Using the credential file you just downloaded, create a Kubernetes Secret and substitute `<keyfile>` with the filename generated by Google for your Google Service Account:
-	```bash
-	kubectl create secret generic workspace-credentials --from-file=workspace-credentials.json=/path/to/<keyfile>.json
-	```
-3. Edit the file located at `scim-examples/beta/workspace-settings.json` and fill in correct values for:
+
+   ```bash
+   kubectl create secret generic workspace-credentials \
+       --from-file=workspace-credentials.json=/path/to/<keyfile>.json
+   ```
+
```

Comment:
Trimming some extra space so the list renders properly.
```suggestion
   ```bash
   kubectl create secret generic workspace-credentials \
       --from-file=workspace-credentials.json=/path/to/<keyfile>.json
   ```
```